homeassistant/components/climate/__init__.py

```diff
@@ -68,8 +69,10 @@
     vol.Required(ATTR_AUX_HEAT): cv.boolean,
 })
 SET_TEMPERATURE_SCHEMA = vol.Schema({
-    vol.Optional(ATTR_ENTITY_ID): cv.entity_ids,
     vol.Required(ATTR_TEMPERATURE): vol.Coerce(float),
+    vol.Inclusive(ATTR_TARGET_TEMP_HIGH, 'auto_mode'): vol.Coerce(float),
+    vol.Inclusive(ATTR_TARGET_TEMP_LOW, 'auto_mode'): vol.Coerce(float),
```

Comment:
I did say the syntax may be off. I'll fiddle with it when I get home tonight

On Sep 5, 2016 12:17 PM, "John Arild Berentsen" <notifications@github.com>
wrote:

> In homeassistant/components/climate/__init__.py
> <https://github.com/home-assistant/home-assistant/pull/3189#discussion_r77549101>
> :
>
> >      vol.Required(ATTR_TEMPERATURE): vol.Coerce(float),
> > +    vol.Inclusive(ATTR_TARGET_TEMP_HIGH, 'auto_mode'): vol.Coerce(float),
> > +    vol.Inclusive(ATTR_TARGET_TEMP_LOW, 'auto_mode'): vol.Coerce(float),
>
> I don't seem to understand the syntax of voluptous. If I put in what you
> suggested, I get Invalid syntax
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/home-assistant/home-assistant/pull/3189/files/c975a1158266846c89904a12a3b1630be316fb72#r77549101>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC2fZd0ZBdUMss_GhxHD_sW3xU_yvOwVks5qnFyogaJpZM4J0nMy>
> .
>